src/faebryk/core/parameter.py

```diff
@@ -1508,6 +1520,10 @@ def int_to_subscript(i: int) -> str:
     def domain_set(self) -> P_Set:
         return self.domain.unbounded(self)
 
+    def to_set(self, solver: "Solver") -> P_Set | None:
+        as_literal = solver.inspect_get_known_supersets(self, force_update=False)
+        return None if as_literal == self.domain_set() else as_literal
+
```

Comment:
I don't like this being here.
But we can leave it here for now.
Rename it to: `get_last_known_deduced_superset`